Cannot build module in a folder with spaces

**Steps:**
* clone the project to a folder with spaces in the name
* try to `yarn`

It fails:

```
CC(target) Release/obj.target/nothing/node_modules/node-addon-api/nothing.o
  LIBTOOL-STATIC Release/nothing.a
warning: /Library/Developer/CommandLineTools/usr/bin/libtool: archive library: Release/nothing.a the table of contents is empty (no object file members in the library define global symbols)
  CXX(target) Release/obj.target/watcher/src/binding.o
clang: error: no such file or directory: 'folder/node_modules/node-addon-api'
make: *** [Release/obj.target/watcher/src/binding.o] Error 1
```

--- a/binding.gyp
+++ b/binding.gyp
@@ -4,7 +4,7 @@
       "target_name": "watcher",
       "defines": [ "NAPI_DISABLE_CPP_EXCEPTIONS" ],
       "sources": [ "src/binding.cc", "src/Watcher.cc", "src/Backend.cc", "src/DirTree.cc" ],
-      "include_dirs" : ["<!@(node -p \"require('node-addon-api').include\")"],
+      "include_dirs" : ["<!@(node -p \"require('node-addon-api').include_dir\")"],
       "dependencies": ["<!(node -p \"require('node-addon-api').gyp\")"],
       'cflags!': [ '-fno-exceptions' ],
       'cflags_cc!': [ '-fno-exceptions' ],

--- a/binding.gyp
+++ b/binding.gyp
@@ -4,7 +4,7 @@
       "target_name": "watcher",
       "defines": [ "NAPI_DISABLE_CPP_EXCEPTIONS" ],
       "sources": [ "src/binding.cc", "src/Watcher.cc", "src/Backend.cc", "src/DirTree.cc" ],
-      "include_dirs" : ["<!@(node -p \"require('node-addon-api').include_dir\")"],
+      "include_dirs" : ["<!(node -p \"require('node-addon-api').include_dir\")"],
       "dependencies": ["<!(node -p \"require('node-addon-api').gyp\")"],
       'cflags!': [ '-fno-exceptions' ],
       'cflags_cc!': [ '-fno-exceptions' ],

